test/test_support.ts

```diff
@@ -40,25 +40,31 @@ export const compilerOptions: ts.CompilerOptions = {
 };
 
 const {cachedLibPath, cachedLib} = (() => {
-  const host = ts.createCompilerHost(compilerOptions);
-  const fn = host.getDefaultLibFileName(compilerOptions);
-  const p = ts.getDefaultLibFilePath(compilerOptions);
+  const host = ts.createCompilerHost(goldenCompilerOptions);
```

Comment:
add a comment why/if it's OK to share these even for calls that don't use the same goldenCompilerOptions?